doc/Command-Reference.md

```diff
@@ -3857,12 +3857,19 @@ This command is used for administratively bringing up the Physical interface or
   admin@sonic:~$ sudo config interface startup Ethernet8,Ethernet16-20,Ethernet32
   ```
 
+**config interface <interface_name> speed (Versions >= 202006)**
+
+Dynamic breakout feature is supported in SONiC from 202006 version.
+User can configure any speed specified under "breakout_modes" keys under the parent interface in the platform-specific port configuration file (i.e. platform.json). Remember, Speed specified in the current breakout mode are the only available speed for the interface at that point of time.
```

Comment:
These two sentences state the same thing in two different ways. I suggest eliminating one of these sentences for brevity and clarity. I would also suggest demonstrating how the supported speeds can be determined i.e.

```
For example for a breakout mode of 2x50G[25G,10G] the default speed is 50G but the interface also supports 25G and 10G.
```

I would also suggest recommending the user to run `show interface breakout current-mode [intf name]` to get the current mode and supported speeds.

Reply:
Modified as requested 👍